.travis.yml

```diff
@@ -27,6 +32,34 @@ script:
 
 jobs:
   include:
+    - stage: Smoke Testing
+      php: 7.3
+      env: DB=sqlite
+
+    - stage: Smoke Testing
+      if: type = pull_request
+      env: DB=none PULL_REQUEST_CODING_STANDARDS
+      php: 7.1
+      install: travis_retry composer install --prefer-dist
+      script:
+        - |
+          if [ $TRAVIS_BRANCH != "master" ]; then
+            git remote set-branches --add origin $TRAVIS_BRANCH;
+            git fetch origin $TRAVIS_BRANCH;
+          fi
+        - git merge-base origin/$TRAVIS_BRANCH $TRAVIS_PULL_REQUEST_SHA || git fetch origin +refs/pull/$TRAVIS_PULL_REQUEST/merge --unshallow
+        - wget https://github.com/diff-sniffer/git/releases/download/0.2.0/git-phpcs.phar
```

Comment:
We can bump this to the latest diff-sniffer/git release (0.3.2) - separate commit would be fine to me but it can be done in another PR if you prefer :+1: